MyGreenGo/amazon-cloudwatch-agent
# feat: add collector v3 config

## Context

We need a way to push and update cloudwatch agent config to collectors

## Proposal

Deploy through Jfrog:

1. Assuming update flow [amazon-cloudwatch-agent-install v1.247358.0](https://connect.jfrog.io/v2/updates/#) has been run against collectors
1. Run update flow Apply [collector v3 config v0.2](https://connect.jfrog.io/v2/updates/#) which will run the `collector-v3-config/setup.sh` as sudo

## Test

Tested on my testbed:
<img width="1308" alt="Screenshot 2023-06-14 at 08 27 59" src="https://github.com/MyGreenGo/amazon-cloudwatch-agent/assets/1225809/8bbd8458-3984-46fe-a273-f47bb978760a">

Procedure:
1. Log in to [jfrog connect](https://connect.jfrog.io/v2/updates/#)
1. go to update flow and create a deployment of `collector v3 config v0.2`  with this PR branch  (`feat/collector-v3-config`) as tag.
1. Check cloudwatch logs for frontend logs
